client/src/client/__init__.py

```diff
@@ -70,12 +70,12 @@ def __init__(self) -> None:
         self.echo_peer_id = (
             "e12dc2da85f995a528d34b4acdc539a720b2bc4912bc1c32c322b201134d3ed6"
         )
-        echo_client = self.db_manager.add_friend(
```

Comment:
Maybe leave adding echo hardcoded for now, so that developers don't need to add it manually every time